### DISCUSSION TITLE
Sleep and add/remove for lists

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#1207] [Archive]
At this point, we have two language requests open:

- #402
- #582 

I love these ideas! Conceptually they fit at level 2 because level 2 introduces lists, but level 2 is quite full already. My proposal would be to add a fresh level 3 for this. I have two competing ideas in mind

## Add all in a new level 3

1. As it is now
2. Introduce lists and variables as it is now
3. Introduce add and remove and sleep
4. What is now 3, etc.

## Add sleep in level 2, all of lists in a new level 3

1. As it is now
2. Introduce only simple variables (like name), also add sleep to make it all more fun
3. Introduce lists with assign as we have it now (dieren is hand, kat, cavia) and also add and remove 
4. What is now 3, etc.

I tend to lean to the second option, also because it balances the adventures more (story, drawing and restaurant can remain at 2 then and the rest (all that uses random) moves to level 3) 

What do you think @MarleenGilsing? Because we now have a program to migrate levels we can easily move the adventures

**Comment by Felienne** [comment]
I just realized we also have https://github.com/Felienne/hedy/issues/885 open. Maybe we can add this into one of the above too?

**Comment by MarleenGilsing** [comment]
I'd like to go for option 2 as well, because variables can be hard to understand and now we teach them really fast. 
Level 2 would then have 3 new options for the students: Using a variable (instead of echo), sleep and name is ask (and turtle options). 
That would be plenty and level 3 will be very entertaining with  the at random command.

**Comment by Felienne** [comment]
Closed by #1316
